docs/content/usage/scenarios/_index.md

```diff
@@ -0,0 +1,29 @@
++++
+title = "Scenarios"
+date = 2020-08-12T12:52:59+03:00
+weight = 3
+chapter = true
+pre = "<i class='fas fa-map-marked-alt'></i> "
++++
+
+# Scenarios
+
+This section describes the different attack <i>Scenarios</i> that the Infection Monkey can simulate.
+
+{{% notice note %}}
+Don't worry! The Infection Monkey uses safe exploiters and does not cause any permanent system modifications that could impact security or operations.
+{{% /notice %}}
+
+The Infection Monkey has pre-built scenarios to simulate common types of attacks that take place. These scenarios, when selected, manipulate the default configuration values and only show the configuration fields relevant to the selected scenario. This makes it possible for you to quickly run the Monkey on your network in order to accomplish a specific objective.
```

Comment:
The technical details of what happens when a user clicks on a scenario are not relevant, they won't help the user to achieve his goals. The purpose of this page is to help the user make a choice of scenario. I would maybe reword to:
"Choose a scenario that best describes your use case. Choosing a "Custom" scenario will allow you to fine-tune your simulation and access some features not available in other scenarios. [Read more about configuring a custom simulation](link-to-custom-page). "

Reply:
WDYT?
![image](https://user-images.githubusercontent.com/44770317/127482506-cc0dd21c-a436-44d0-96e6-58e768400a02.png)